### ISSUE TITLE
Windows Compatibility

### ISSUE THREAD

**Issue by saadmanrafat** [alichtman/shallow-backup#77]
I was wondering if this could be extended to work on Windows.

**Comment by alichtman** [comment]
Yeah, this isn't limited to just macOS, but it would require a bunch of refactoring and testing that just isn't a priority of mine. I'm definitely open to a PR, though.

**Comment by alichtman** [comment]
**Implementation Plan**

- [ ] Use `os.path.join` obsessively, for all paths

Is this it?

**Comment by saadmanrafat** [comment]
Yes but another issue is inquirer, which doesn't work on Windows. Other viable choices are prompt_toolkit and Pyinquirer. Each has it's pros and cons. But using any one of them could lead to losing some features the repo already supports like the List prompt.

**Comment by alichtman** [comment]
One possibility is forking it and maintaining a Windows version separately.

**Comment by alichtman** [comment]
Another possibility is `try` / `except` importing `inquirer` and if that fails, import some other library that's Windows compatible. This is probably the best solution, as it would avoid forking the project. I still don't use Windows, though, so I won't be fixing this.

**Comment by saadmanrafat** [comment]
try except might work. I'll look into it.
